docs/articles/documentation/using-testcafe/common-concepts/test-https-and-http2-websites.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: docs
+title: Test HTTPS and HTTP/2 Websites
+permalink: /documentation/using-testcafe/common-concepts/test-https-and-http2-websites.html
+redirect_from:
+  - /documentation/using-testcafe/common-concepts/connect-to-the-testcafe-server-over-https.html
+---
+# Test HTTPS and HTTP/2 Websites
+
+This topic describes how TestCafe works with HTTPS and HTTP/2 websites.
+
+* [Test HTTPS Websites](#test-https-websites)
+* [Test HTTP/2 Websites](#test-http2-websites)
+
+## Test HTTPS Websites
+
+TestCafe is a proxy-based testing tool. The TestCafe [reverse proxy](https://en.wikipedia.org/wiki/Reverse_proxy) serves the tested webpage over HTTP and communicates with the original web server over HTTP or HTTPS (depends on the [specified page URL](../../test-api/test-code-structure.md#specifying-the-start-webpage)).
```

Comment:
Change "depends on the [specified page URL]" to "depending on the [specified page URL]"